<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Group by תרגיל 110 Assertion error</title>
    <link>https://forums.pythonic.guru/t/group-by-110-assertion-error/6791</link>
    <description>ניסיתי לעבור על הודעות קודמות בנושא הודעות הבודק האוטומטי אבל לא הצלחתי מלהבין מה לא בסדר ומה צריך לתקן...
![image|690x59](upload://lSMEf2914GMMgzclNNuz9yF9XQY.png) 

![image|689x44](upload://yUQ9LtcaNmkiUjlAblly99KrbR0.png) 

תודה על העזרה...</description>
    <language>he</language>
    <lastBuildDate>Fri, 29 May 2020 13:57:27 +0000</lastBuildDate>
    <category>שבוע 6</category>
    <atom:link href="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Group by תרגיל 110 Assertion error</title>
        <dc:creator><![CDATA[benhamou.l.e]]></dc:creator>
        <description><![CDATA[
            <p>תודה, הכן ביקשתי מהפונקציה להדפיס במקום להחזיר…<br>
הפלט היה בסדר, אז לא הבנתי ראשית מה הבעיה… תודה !</p>
          <p><a href="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/3">קריאת הנושא במלואו</a></p>
        ]]></description>
        <link>https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/3</link>
        <pubDate>Fri, 29 May 2020 13:57:27 +0000</pubDate>
        <guid isPermaLink="false">forums.pythonic.guru-post-6791-3</guid>
        <source url="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791.rss">Group by תרגיל 110 Assertion error</source>
      </item>
      <item>
        <title>Group by תרגיל 110 Assertion error</title>
        <dc:creator><![CDATA[yuval.cagan]]></dc:creator>
        <description><![CDATA[
            <p>הבודק האוטומטי משווה את התוצאה שמחזירה הפונקציה שלך עם התוצאה הצפויה.<br>
נראה שהפונקציה שלך מחזירה None.</p>
          <p><a href="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/2">קריאת הנושא במלואו</a></p>
        ]]></description>
        <link>https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/2</link>
        <pubDate>Fri, 29 May 2020 13:35:50 +0000</pubDate>
        <guid isPermaLink="false">forums.pythonic.guru-post-6791-2</guid>
        <source url="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791.rss">Group by תרגיל 110 Assertion error</source>
      </item>
      <item>
        <title>Group by תרגיל 110 Assertion error</title>
        <dc:creator><![CDATA[benhamou.l.e]]></dc:creator>
        <description><![CDATA[
            <p>ניסיתי לעבור על הודעות קודמות בנושא הודעות הבודק האוטומטי אבל לא הצלחתי מלהבין מה לא בסדר ומה צריך לתקן…<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://forums.pythonic.guru/uploads/default/original/2X/9/995f071d05744e03d2789d775f36c3bd457a2488.png" data-download-href="https://forums.pythonic.guru/uploads/default/995f071d05744e03d2789d775f36c3bd457a2488" title="image"><img src="https://forums.pythonic.guru/uploads/default/original/2X/9/995f071d05744e03d2789d775f36c3bd457a2488.png" alt="image" data-base62-sha1="lSMEf2914GMMgzclNNuz9yF9XQY" width="690" height="59" data-small-upload="https://forums.pythonic.guru/uploads/default/optimized/2X/9/995f071d05744e03d2789d775f36c3bd457a2488_2_10x10.png"></a></div><p></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://forums.pythonic.guru/uploads/default/original/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a.png" data-download-href="https://forums.pythonic.guru/uploads/default/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a" title="image"><img src="https://forums.pythonic.guru/uploads/default/optimized/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a_2_689x44.png" alt="image" data-base62-sha1="yUQ9LtcaNmkiUjlAblly99KrbR0" width="689" height="44" srcset="https://forums.pythonic.guru/uploads/default/optimized/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a_2_689x44.png, https://forums.pythonic.guru/uploads/default/optimized/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a_2_1033x66.png 1.5x, https://forums.pythonic.guru/uploads/default/original/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a.png 2x" data-small-upload="https://forums.pythonic.guru/uploads/default/optimized/2X/f/f4b6cdb30d22a22c0e53a2d45468db05d3cc702a_2_10x10.png"></a></div><p></p>
<p>תודה על העזרה…</p>
          <p><a href="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/1">קריאת הנושא במלואו</a></p>
        ]]></description>
        <link>https://forums.pythonic.guru/t/group-by-110-assertion-error/6791/1</link>
        <pubDate>Fri, 29 May 2020 13:33:59 +0000</pubDate>
        <guid isPermaLink="false">forums.pythonic.guru-post-6791-1</guid>
        <source url="https://forums.pythonic.guru/t/group-by-110-assertion-error/6791.rss">Group by תרגיל 110 Assertion error</source>
      </item>
  </channel>
</rss>
